internal/pkg/describe/uri.go

```diff
@@ -282,23 +283,22 @@ func (d *uriDescriber) uri() (accessURI, error) {
 		return accessURI{}, fmt.Errorf("get stack resources for service %s: %w", d.svc, err)
 	}
 
-	var ruleARN string
+	var ruleARNs []string
 	for _, resource := range svcResources {
 		if resource.Type == svcStackResourceListenerRuleResourceType &&
-			((httpsEnabled && resource.LogicalID == svcStackResourceHTTPSListenerRuleLogicalID) ||
-				(!httpsEnabled && resource.LogicalID == svcStackResourceHTTPListenerRuleLogicalID)) {
-			ruleARN = resource.PhysicalID
-			break
+			(httpsEnabled && strings.HasPrefix(resource.LogicalID, svcStackResourceHTTPSListenerRuleLogicalID)) ||
+			(!httpsEnabled && strings.HasPrefix(resource.LogicalID, svcStackResourceHTTPListenerRuleLogicalID)) {
+			ruleARNs = append(ruleARNs, resource.PhysicalID)
```

Comment:
Was this a bug previously? The fact that we only use the first matched listener rule arn.

Reply:
yeah that's true. we log only the URI of only `HTTPSLISTENERRULE` but not URI in `HTTPSLISTENERRULE1`